src/main/java/org/pkl/spring/boot/PklAutoConfiguration.java

```diff
@@ -17,15 +17,15 @@
 
 import java.util.*;
 import org.pkl.core.PNull;
+import org.springframework.boot.autoconfigure.AutoConfiguration;
 import org.springframework.boot.context.properties.ConfigurationPropertiesBinding;
-import org.springframework.context.annotation.Configuration;
 import org.springframework.core.convert.TypeDescriptor;
 import org.springframework.core.convert.converter.GenericConverter;
 import org.springframework.core.env.ConfigurableEnvironment;
 import org.springframework.lang.Nullable;
 import org.springframework.stereotype.Component;
 
-@Configuration
+@AutoConfiguration
```

Comment:
What does this mean for Spring Boot 2 users? Are there any compatibility issues here?